blimflix/client/app/all/all.controller.js

```diff
@@ -1,53 +1,23 @@
 (function() {
   'use strict';
 
-  /**
-   * Chain to fetch module
-   *   https://github.com/johnpapa/angular-styleguide#style-y022
-   */
   angular
     .module('blimFlix.all')
     .controller('AllController', AllController);
 
-  /**
-   * Avoid anonymous functions as callbacks
-   *   https://github.com/johnpapa/angular-styleguide#style-y024
-   *
-   * Document dependency injenction using annotations
-   *  https://github.com/johnpapa/angular-styleguide#style-y100
-   */
-  /* @ngInject */
-  function AllController() {
-
-    /**
-     * Attach any view properties to this variable
-     *   https://github.com/johnpapa/angular-styleguide#style-y032
-     */
-    /* jshint validthis: true */
+    function AllController() {
     var vm = this;
+    vm.data={
+      name:"sara",
+      team:"beatles"
+    }
```

Comment:
Actually I think that I should remove or comment that, because it was just for the test that we were doing, what do you think?